config/locales/gobierto_dashboards/views/ca.yml

```diff
@@ -44,11 +43,45 @@ ca:
           tenders_for: licitacions per un import de
         tender_amount: Imp. licitació
         title: Contractes i licitacions
+      subsidies:
+        amount: Import
+        amount_distribution: Distribució per imports
+        beneficiary: beneficiari
+        category: Categoria
+        date: Data
+        dates: Dates
+        description: Visualització i anàlisi de les subvencions concedides per %{entity_name}
+        fromto: De a Nº Subv.
+        main_beneficiaries: Principals beneficiaris
+        more: Més de
+        nav:
+          subsidies: Subvencions
+          summary: Resum
+        regulatory_bases: Bases reguladores
+        subsidies: Subvencions
+        summary:
+          collective_subsidies_for: a col·lectius per import de
+          individual_subsidies_for: a particulars per import de
+          label_half_spendings_subsidies: El <strong> <span id = "half-spendings-subsidies-pct">
+            </ span> </ strong> de subvencions concentren el 50% de tota la despesa
+          label_larger_subsidy_amount: La major subvenció suposa <strong> <span id
+            = "larger-subsidy-amount-pct"> </ span> </ strong> de tota la despesa
+            en subvencions
+          label_less_than_1000: El <strong> <span id = "less-than-1000-pct"> </ span>
+            </ strong> de les subvencions són menors de <strong> 1.000 € </ strong>
+          mean_amount: Import mitjà
+          median_amount: Import mitjà
```

Comment:
You cannot use the same label for different things, median (maths) is mediano/a

Reply:
I know mean and median are different, but don't they use the same word in catalan? If not, which is it?

Reply:
Yes, they use the same words :D but there are synomis. You can use mitja for mean and mitjana for median

Reply:
👍 I'll change it

Reply:
https://ca.wikipedia.org/wiki/Mediana